README.md

```diff
@@ -46,6 +46,7 @@ So we need to install these packages first:
     # Ubuntu 10.4+
     sudo apt-get install protobuf-c-compiler
     sudo apt-get install libprotobuf-c0-dev
+    sudo apt-get install libprotoc-dev
```

Comment:
Are you positive that it's `libprotoc-dev` we need and not `libprotobuf-dev`, a transitive dependency thereof?